src/contexts/wallet.js

```diff
@@ -76,6 +94,7 @@ function WalletContextProvider({ children }) {
       status,
       chainId: connected ? chainId : 1, // connect to mainnet if wallet is not connected
       connected,
+      changeNetworkTypeDisconnected: changeNetworkTypeDisconnected,
       ...walletBaseRest,
```

Comment:
can just use `changeNetworkTypeDisconnected,` here.

Reply:
When DAO-269 is implemented, these lines should be replaced with the search result. So, I'm good to leave these lines as-is for this PR.